index.html

```diff
@@ -1,28 +1,197 @@
+            <h3 class="mb-0">Framework</h3>
+          </div>
+          <p class="mb-0">
+            learning popular JavaScript frameworks and libraries such as React,
+            Vue.js, and Angular, which help development of web applications.
+          </p>
+        </div>
+        <div class="program d-flex flex-row flex-md-column">
+          <div
+            class="col-4 d-flex flex-column justify-content-center align-items-center"
+          >
+            <i class="fa-sharp fa-solid fa-server"></i>
+            <h3 class="mb-0">ServerSide</h3>
+          </div>
+          <p class="mb-0">
+            Using JavaScript to build server-side applications using Node.js,
+            which allows JavaScript tofull-stack web applications.
+          </p>
+        </div>
+        <div class="program d-flex flex-row flex-md-column">
+          <div
+            class="col-4 d-flex flex-column justify-content-center align-items-center"
+          >
+            <i class="fa-solid fa-bug"></i>
+            <h3 class="mb-0">Debugging</h3>
+          </div>
+          <p class="mb-0">
+            It involves analyzing the code to determine the source of the
+            problem and making changes to fix it. Using debuggers or logging
+            tools to track the program's execution.
+          </p>
+        </div>
+        <div class="program d-flex flex-row flex-md-column">
+          <div
+            class="col-4 d-flex flex-column justify-content-center align-items-center"
+          >
+            <i class="fa-sharp fa-solid fa-diagram-project"></i>
+            <h3 class="mb-0">Projects</h3>
+          </div>
+          <p class="mb-0">
+            Completing hands-on projects to apply the skills learned throughout
+            the course and build a portfolio of web development projects.
+          </p>
+        </div>
+      </div>
+      <br />
+      <button class="enrol">Enroll now</button>
+      <h5>
+        And take the first step towards a rewarding career in web development!"
+      </h5>
+    </section>
 
+    <section class="speak" id="speaker">
+      <h2>Feature Professors</h2>
+      <hr class="w-50 text-light" />
+      <div class="speaker d-flex flex-column flex-md-row" id="prof"></div>
+      <button
+        class="d-md-none w-50 d-flex justify-content-center align-items-center"
+      >
+        <h3 class="mb-0">More</h3>
+        <i class="fa-solid fa-chevron-down text-danger"></i>
+      </button>
+    </section>
 
+    <section class="partner">
+      <h2 class="text-center">Our partner</h2>
+      <hr />
+      <div class="d-flex flex-wrap gap-2 part-name">
+        <h3>Mozilla</h3>
+        <h3>Google</h3>
+        <h3>Aple</h3>
+        <h3>Uber</h3>
+        <h3>Amazone</h3>
+      </div>
+    </section>
 
+    <footer>
+      <i class="fa-brands fa-square-js"> <span>Full course</span> </i>
+      <i class="fa-sharp fa-regular fa-copyright"> <span>2023</span> </i>
```

Comment:
- [ ] Kindly increase the padding around elements in your footer section and adjust the background color to match the original design. The same also applies to the footer in your `about me` section

![image](https://user-images.githubusercontent.com/98453979/225684688-569aaf54-ef0e-45f4-b9cc-27a676580731.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/225684945-ff6823b4-7c93-471c-8488-620206d8262f.png)

Reply:
Your are right @frank1738